.gitignore

```diff
@@ -2,7 +2,6 @@
 .idea
 .vscode
 .idea
-dist
```

Comment:
Please don't commit `dist` folder to the repository. Automatically generated code should not be commited. Just like `node_modules` :)